_posts/2021-05-11-row_pattern_matching.md

```diff
@@ -0,0 +1,468 @@
+---
+layout: post
+title:  "Row pattern recognition via MATCH_RECOGNIZE"
+author: Kasia Findeisen (kasiafi)
+excerpt_separator: <!--more-->
+---
+
+The `MATCH_RECOGNIZE` syntax was introduced in the latest SQL specification
+of 2016. It is a super powerful tool for analyzing trends in your data. You can
+define a pattern using the well-known regular expression syntax, and match it
+to a set of rows. Upon finding a matching row sequence, you can retrieve all
+kinds of detailed or summary information about the match, and pass it on to be
+processed by the subsequent parts of your query. This is a new level of what a
+pure SQL statement can do. We are proud to announce that Trino supports this
```

Comment:
Do you mean the sentence "We are proud ..."? I intended to start with a short description and then mention Trino. I can't find a way to restructure it.

Reply:
The `MATCH_RECOGNIZE` syntax was introduced in the latest SQL specificationof 2016. It is a super powerful tool for analyzing trends in your data.  We are proud to announce that Trino supports thisgreat feature since [version 356]({{site.url}}/docs/current/release/release-356.html). You can define ...